.github/workflows/builds.yml

```diff
@@ -248,8 +248,7 @@ jobs:
         include:
           - image: ubuntu2204
```

Comment:
Should we add back the clang build using the new image that includes clang as well?

Reply:
Done!